packages/paymaster/src/BiconomyPaymaster.ts

```diff
@@ -75,31 +93,23 @@ export class BiconomyPaymaster implements IHybridPaymaster {
     */
```

Comment:
Will you add this custom token handling in later PR? Coz we clearly know in case of USDT it needs to be handled separately. There should be a separate helper class that should have this custom logic of setting 0 token approval first based of list of tokens per network.

Reply:
yes planning to do this as separate feature. cause i have checked only on ethereum mainnet USDT contract behaves like this, not for polygon or BNB.